src/test/java/com/wikia/webdriver/common/logging/VelocityWrapper.java

```diff
@@ -0,0 +1,179 @@
+    VelocityContext context = new VelocityContext();
+
+    context.put("className", "\"" + classList.stream().collect(Collectors.joining(" ")) + "\"");
+    context.put("command", command);
+    context.put("escapedDescription", description);
+    StringWriter writer = new StringWriter();
+    t.merge(context, writer);
+    builder.append(writer.toString());
+    return builder.toString();
+//    CommonUtils.appendTextToFile(logPath, builder.toString());
+  }
+
+  public static String appendLastLogRow() {
+    VelocityEngine ve = new VelocityEngine();
+    StringBuilder builder = new StringBuilder();
+
+    Template t =
+        ve.getTemplate("./src/test/java/com/wikia/webdriver/common/velocitytemplates/lastLogRow.vm");
+    VelocityContext context = new VelocityContext();
+
+    StringWriter writer = new StringWriter();
+    t.merge(context, writer);
+    builder.append(writer.toString());
+    return builder.toString();
+//    CommonUtils.appendTextToFile(logPath, builder.toString());
+  }
+
+  public static String appendLogRowWithLink(String link, String label) {
+    VelocityEngine ve = new VelocityEngine();
+    StringBuilder builder = new StringBuilder();
+
+    Template t =
+        ve.getTemplate("./src/test/java/com/wikia/webdriver/common/velocitytemplates/logRowWithLink.vm");
+    VelocityContext context = new VelocityContext();
+    context.put("link", link);
+    context.put("label", label);
+    StringWriter writer = new StringWriter();
+    t.merge(context, writer);
+    builder.append(writer.toString());
+    return builder.toString();
+//    CommonUtils.appendTextToFile(logPath, builder.toString());
+  }
+
+  public static String appendImage(String imageAsBase64) {
+    VelocityEngine ve = new VelocityEngine();
+    StringBuilder builder = new StringBuilder();
+
+    Template t =
+        ve.getTemplate("./src/test/java/com/wikia/webdriver/common/velocitytemplates/image.vm");
+    VelocityContext context = new VelocityContext();
+
+    context.put("imageAsBase64", imageAsBase64);
+    StringWriter writer = new StringWriter();
+    t.merge(context, writer);
+    builder.append(writer.toString());
+    return builder.toString();
+  }
+
+  public static String appendErrorLogRow(List<String> classList, String command, String imageCounter) {
+    VelocityEngine ve = new VelocityEngine();
+    StringBuilder builder = new StringBuilder();
+
+    Template t =
+        ve.getTemplate(
+            "./src/test/java/com/wikia/webdriver/common/velocitytemplates/errorLogRow.vm");
+    VelocityContext context = new VelocityContext();
+    context.put("className", "\"" + classList.stream().collect(Collectors.joining(" ")) + "\"");
+    context.put("command", command);
+    context.put("imageCounter", imageCounter);
+    StringWriter writer = new StringWriter();
+    t.merge(context, writer);
+    builder.append(writer.toString());
+    return builder.toString();
+//    CommonUtils.appendTextToFile(logPath, builder.toString());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)